source/_integrations/image.markdown

```diff
@@ -8,6 +8,7 @@ ha_codeowners:
   - '@home-assistant/core'
 ha_domain: image
 ha_quality_scale: internal
+ha_iot_class: Calculated
```

Comment:
This integration provides no communication or any entities. Therefore it has no IoT class

```suggestion
ha_iot_class: ~
```